### PR TITLE
new layout for "Summary Translated" by Team issue #324

### PR DESCRIPTION
**Pull request by S-mardii**
Update layout of Summary Translated by Team

### CODE REVIEW

File: inc/language-manager.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ function init_language_manager(){
 
   function get_the_language_by_language_code($lang_code = 'en')
   {
-      return $supported_languages[$lang_code];
+      return $this->supported_languages[$lang_code];
```
Comment (acorbi) [inline inc/language-manager.php]: @S-mardii well spotted!

Comment (S-mardii) [inline inc/language-manager.php]: Credit to Huyeng :)

File: inc/templates/post-list-single-1-cols.php [hunk 2]
```diff
@@ -44,9 +44,9 @@
 			<?php echo_post_meta($post,array('date','sources'),$order); ?>
 		<?php endif; ?>
 
-		<?php if ($show_summary_translated_by_odc_team): ?>
+		<!-- <?php if ($show_summary_translated_by_odc_team): ?>
```
Comment (acorbi) [inline inc/templates/post-list-single-1-cols.php]: @S-mardii please do not leave commented code behind. If we do not need this anymore then it should be removed.

File: inc/language-manager.php (hunk 1 above)
Comment (acorbi) [inline inc/language-manager.php]: @Huyeng you are the best :tada:

File: inc/templates/post-list-single-2-cols.php [hunk 3]
```diff
@@ -41,12 +41,13 @@
 		<?php endif; ?>
 
 		<?php if ($show_meta): ?>
-			<?php echo_post_meta($post,array('date','sources'),$order); ?>
+			<?php echo_post_meta($post,array('date','sources','show_summary_translated_by_odc_team'),$order); ?>
 		<?php endif; ?>
 
-		<?php if ($show_summary_translated_by_odc_team): ?>
+		<!-- <?php if ($show_summary_translated_by_odc_team): ?>
```
Comment (acorbi) [inline inc/templates/post-list-single-2-cols.php]: @S-mardii please do not leave commented code behind. If we do not need this anymore then it should be removed.

File: inc/templates/post-list-single-4-cols.php [hunk 4]
```diff
@@ -44,9 +44,9 @@
 			<?php echo_post_meta($post,array('date','sources'),$order); ?>
 		<?php endif; ?>
 
-		<?php if ($show_summary_translated_by_odc_team): ?>
+		<!-- <?php if ($show_summary_translated_by_odc_team): ?>
```
Comment (acorbi) [inline inc/templates/post-list-single-4-cols.php]: @S-mardii please do not leave commented code behind. If we do not need this anymore then it should be removed.

Comment (S-mardii) [inline inc/templates/post-list-single-4-cols.php]: noted

**Comment by acorbi** [comment]
@S-mardii thanks for attending my comments. merging...

**Pull request merged**